app/controllers/issue_templates_controller.rb

```diff
@@ -1,206 +1,208 @@
+
+    unless project_default_template.blank?
+      default_template = project_default_template.id
+    end
+
+    unless issue_templates.empty?
+      issue_templates.each { |x| group.push([x.title, x.id]) }
+    end
+
+    if inherit_template
+=begin
+      project_ids.each do |i|
+        inherit_templates.concat(
+            IssueTemplate.search_by_project(i)
+                .search_by_tracker(@tracker.id)
+                .enabled.enabled_sharing
+                .order_by_position
+        )
+      end
+=end
+      inherit_templates = get_inherit_templates(project_ids, @tracker.id)
+
+      if inherit_templates.any?
+        inherit_templates.each do |x|
+          group.push([x.title, x.id, { class: 'inherited' }])
+          next unless x.is_default == true
+          default_template = x if project_default_template.blank?
+        end
+      end
+    end
+
+    global_issue_templates = GlobalIssueTemplate.joins(:projects)
+                                 .search_by_tracker(@tracker.id)
+                                 .search_by_project(@project.id)
+                                 .order_by_position
+
+    if global_issue_templates.any?
+      global_issue_templates.each do |x|
+        group.push([x.title, x.id, { class: 'global' }])
+      end
+    end
+
+    is_triggered_by_status = request.parameters[:is_triggered_by_status]
+    grouped_options.push([@tracker.name, group]) if group.any?
+    render action: '_template_pulldown', layout: false,
+           locals: { is_triggered_by_status: is_triggered_by_status, grouped_options: grouped_options,
+                     should_replaced: setting.should_replaced, default_template: default_template }
+  end
+
+  # preview
+  # @return [Object]
+  def preview
+    @text = (params[:issue_template] ? params[:issue_template][:description] : nil)
+    @issue_template = IssueTemplate.find(params[:id]) if params[:id]
+    render partial: 'common/preview'
+  end
+
+  # Reorder templates
+  def move
+    move_order(params[:to])
+  end
+
+  private
+
+  def find_user
+    @user = User.current
+  end
+
+  def find_tracker
+    @tracker = Tracker.find(params[:issue_tracker_id])
+  end
+
+  def find_object
+    @issue_template = IssueTemplate.find(params[:id])
+    @project = @issue_template.project
+  rescue ActiveRecord::RecordNotFound
+    render_404
+  end
+
+  def find_project
+    @project = Project.find(params[:project_id])
+  rescue ActiveRecord::RecordNotFound
+    render_404
+  end
+
+  def move_order(method)
+    IssueTemplate.find(params[:id]).send "move_#{method}"
+    respond_to do |format|
+      format.html { redirect_to action: 'index' }
+      format.xml  { head :ok }
+    end
+  end
+
+  def get_inherit_templates(project_ids, tracker_id)
+    # keep ordering of project tree
+    # TODO: Add Test code.
+    inherit_templates = []
+    project_ids.each do |i|
+      inherit_templates.concat(IssueTemplate.search_by_project(i)
```

Comment:
__[rubocop]__
* Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.